### DISCUSSION TITLE
Why doesn't my inbox show notifications that mention me?

### DISCUSSION THREAD

**Discussion by SageJustus** [community/community#136541] [New to GitHub] [answered]
Here's my inbox, it's empty.
![image](https://github.com/user-attachments/assets/bc203355-8dc3-427d-bbd2-256268068a87)

But I found someone mentioned me in a [discussion](https://github.com/qos-ch/logback/discussions/846).
![image](https://github.com/user-attachments/assets/c9b55b25-0934-4e41-bca5-2bbabd8a331d)

What did I miss?

**Comment by davevad93** [comment, marked as answer]
Hi @SageJustus ,

Check your [notification settings](https://github.com/settings/notifications), scroll down to the `Subscriptions` section and make sure you receive notifications on GitHub.

![000](https://github.com/user-attachments/assets/00f6313f-85cf-461b-a9a8-48dbd495986f)

**Reply by SageJustus** [reply]
Thanks, I'll try and see if this works. Can you mention me again?

**Reply by davevad93** [reply]
You're welcome @SageJustus , can you receive notifications in your inbox now?

**Reply by SageJustus** [reply]
Yes, It's working.

**Reply by ghostinhershell** [reply]
Thanks so much for helping here @davevad93!
